ssv/spectest/tests/runner/duties/proposer/propose_blinded_decide_regular.go

```diff
@@ -28,9 +28,8 @@ func ProposeBlindedBlockDecidedRegular() tests.SpecTest {
 					[]types.OperatorID{1, 2, 3},
 					qbft.Height(testingutils.TestingDutySlotV(spec.DataVersionBellatrix)),
 					testingutils.ProposerMsgID,
-					testingutils.TestProposerConsensusDataBytsV(spec.DataVersionBellatrix),
-				),
-				nil),
+					testingutils.TestProposerConsensusDataBytsV(ks, spec.DataVersionBellatrix),
```

Comment:
we have a ton of those changes which just add "changed" files to the PR

Reply:
Many of them come from your commit
0266dd81

I'll try to clean it up